packages/app-desktop/gui/MainScreen/commands/notesSortOrderSwitch.ts

```diff
@@ -0,0 +1,94 @@
+import { CommandContext, CommandDeclaration, CommandRuntime } from '@joplin/lib/services/CommandService';
+import Setting from '@joplin/lib/models/Setting';
+import { _ } from '@joplin/lib/locale';
+
+export const NOTES_SORT_ORDER_SWITCH = 'notesSortOrderSwitch';
+export const SETTING_FIELD = 'notes.sortOrder.field';
+export const SETTING_REVERSE = 'notes.sortOrder.reverse';
+export const SETTING_PER_FIELD_REVERSAL_ENABLED = 'notes.perFieldReversalEnabled';
+export const SETTING_PER_FIELD_REVERSE = 'notes.perFieldReverse';
```

Comment:
Please don't use constants for setting name as it's another layer of indirection, just use the string directly.

Reply:
finished. 4cdba95